packages/.eslintrc.json

```diff
@@ -1,197 +0,0 @@
-{
-    "root": true,
```

Comment:
I pulled these out into the root config but this could use a double check and a validation to make sure nothing was lost in the process.